tests/commands.guild.prefix.test.js

```diff
@@ -0,0 +1,15 @@
+import { dispatchPrefixFun } from "../lib/commands/guild/prefix";
+
+it("should throw an error, because of too long prefix", async () => {
+  const result = await dispatchPrefixFun("1", ["1234"]).catch(e => {
+    expect(e).toBeInstanceOf(Error);
+  });
+  expect(result).toBeUndefined();
+});
+
+it("should throw an error, because of too long array", async () => {
+  const result = await dispatchPrefixFun("1", ["!", "%"]).catch(e => {
+    expect(e).toBeInstanceOf(Error);
+  });
+  expect(result).toBeUndefined();
+});
```

Comment:
Maybe also a test that changes the prefix and does not result in an error?

Reply:
Yes, I wanted to, but we need a mock database first. I wasn't sure, @Xol is working on it. Otherwise I could build an interface to decouple the database from the function even further, but that seems a bit much.